### ISSUE TITLE
[Feature request] Normal spectrogram

### ISSUE THREAD

**Issue by ShawkMusic** [rbn42/panon#106]
The current spectrogram has a weird tilt, a blur effect, and the octaves aren't separated equally

I like to do sound design and analysis and it would be really nice to have a spectrogram that doesn't have any of those weird properties. I tried modifying the code for it myself but I wasn't sure what to change. If someone could at least tell me what to change in the code to remove those effects so that I could remove it for myself, that would be nice.

**Comment by archfemboy** [comment]
its just widget for desktop, why dont u use pro software for this?

**Comment by undera** [comment]
Yes, it's a widget for desktop, so it's crucial to have control over something that you see for so long during your day.

**Comment by ShawkMusic** [comment]
> its just widget for desktop, why dont u use pro software for this?

Because it's really convenient to be able to see the spectrogram quickly and easily at any time at a moments notice just by glancing at the edge of my screen

And besides, even just as a desktop decoration, aesthetically the tilt and the blur is kinda ugly
